src/components/TopRatedMovie.tsx

```diff
@@ -0,0 +1,38 @@
+import { TopRatedProps } from '@/lib/types';
+import Image from 'next/image';
+import Play from './layout/Play';
+import Link from 'next/link';
+
+export default function TopRatedMovies({ fetchData }: TopRatedProps) {
+  const top10Movies = fetchData.results.slice(0, 10);
+  const randomIndex = Math.floor(Math.random() * top10Movies.length);
+  const movie = top10Movies[randomIndex];
+  
+  return (
+    <div className="w-full">
+      <div className="text-center relative w-full h-[415px] overflow-hidden">
+        <Link href={`/detail/${movie.id}`}>
+          <div className="relative w-full h-full cursor-pointer block">
+            <Image
+              src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`}
+              fill
+              style={{ objectFit: 'cover' }}
+              alt={movie.title}
+              className="rounded-md"
+            />
+            <div className="absolute inset-0 bg-gradient-to-b from-black via-transparent to-black"></div>
```

Comment:
요 스타일 컴포넌트로 생성해 주시는게 가독성에도 좋고 MovieDetail.tsx 에서도 같은 코드가 사용되고 있어 재사용성에도 좋을 것 같아요!!

```suggestion
   <GradientStyle />
```